src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets

```diff
@@ -267,16 +267,73 @@ Copyright (c) .NET Foundation. All rights reserved.
     <AdditionalProbingPath Include="$(UserProfileRuntimeStorePath)$([System.IO.Path]::DirectorySeparatorChar)|arch|$([System.IO.Path]::DirectorySeparatorChar)|tfm|" />
   </ItemGroup>
 
+  <PropertyGroup>
+    <CompileDependsOn>
+      $(CompileDependsOn)
+      _CreateAppHost;
```

Comment:
Sanity check: this file gets imported _after_ `Microsoft.Common.CurrentVersion.targets`, right?

Reply:
Right.